ui/src/app/edge/history/storage/chargerchart.component.ts

```diff
@@ -19,7 +19,7 @@ export class StorageChargerChartComponent extends AbstractHistoryChart implement
 
     ngOnChanges() {
         this.updateChart();
-    };
+    }
```

Comment:
this change is weird

Reply:
having no semi-colons at the end of function body is recommended??

Reply:
in JavaScript syntax, Function Definetion is not included semicoron.
This semicoron is a Empty Statement.
Perhaps This semi is typo.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/Empty